src/pages/dashboards/DashboardList.tsx

```diff
@@ -38,22 +96,40 @@ const DashboardList = () => {
             ))}
           </ul>
 
-          <div className='mt-16 mb-24 flex items-center justify-end tablet:mb-48'>
-            <span className='mr-16 text-xs font-normal tablet:text-md'>1 페이지 중 1</span>
-            <Pagination currentPage={1} totalPages={4} onPageChange={() => {}} />
-          </div>
+          {totalDashboardPage > 1 && (
+            <div className='mt-16 mb-24 flex items-center justify-end tablet:mb-48 pc:-mb-34'>
+              <span className='mr-16 text-xs font-normal tablet:text-md'>
+                {page} 페이지 중 {totalDashboardPage}
```

Comment:
```suggestion
                {totalDashboardpage} 페이지 중 {page}
```
이게 맞지 않나요..? (3 페이지 중 1)

Reply:
수정했습니다!